src/Log4Net/NewRelic.LogEnrichers.Log4Net/NewRelic.LogEnrichers.Log4Net.csproj

```diff
@@ -41,6 +41,7 @@
   <ItemGroup>
     <PackageReference Include="log4net" Version="2.0.8" />
     <PackageReference Include="NewRelic.Agent.Api" Version="8.20.262" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
```

Comment:
Just adding a comment here to explain that this version was chosen as the minimum version based on this version fixing some serialization issues we experienced on Linux.